cmake/Dependencies.cmake

```diff
@@ -1275,6 +1275,7 @@ endif()
 if(NOT INTERN_BUILD_MOBILE)
   set(TORCH_CUDA_ARCH_LIST $ENV{TORCH_CUDA_ARCH_LIST})
   set(TORCH_NVCC_FLAGS $ENV{TORCH_NVCC_FLAGS})
+  separate_arguments(TORCH_NVCC_FLAGS)
```

Comment:
Is this related to this specific PR or is this more of just code cleanup?

Reply:
It fixes the bug when one passes `TORCH_NVCC_FLAGS` to separately linked binary (in cmake world arguments in the list have to be separated by semicolon rather than list)